### DISCUSSION TITLE
Dịch slack variable

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#36] [Terminology] [answered]
Từ này ở phần SVM https://github.com/mlbvn/handson-ml2-book/pull/138/files#r580355836 đang để là `biến bù`.
Mọi người cho ý kiến nhé @mlbvn/handson-ml2

**Comment by quangvnai** [comment, marked as answer]
Chắc là ok, thấy trong [tài liệu tối ưu hóa](http://dulieu.tailieuhoctap.vn/books/giao-duc-dai-cuong/toan-roi-rac/file_goc_769150.pdf) có sử dụng cách dịch này.
